tests/lava/lib/dl/netx/test_blocks.py

```diff
@@ -231,3 +222,128 @@ def test_dense(self) -> None:
             f'Found {s[s != s_gt] = } and {s_gt[s != s_gt] = }. '
             f'Error was {s_error}.'
         )
```

Comment:
The following block seems to extend the scope of the PR, that's Ok but check you intended these tests to go in.